src/c++/perf_analyzer/test_command_line_parser.cc

```diff
@@ -1210,7 +1267,7 @@ TEST_CASE("Testing Command Line Parser")
 
     CheckValidRange(
         args, option_name, parser, act, exp->is_using_periodic_concurrency_mode,
-        exp->periodic_concurrency_range);
+        exp->periodic_concurrency_range, exp);
```

Comment:
Aren't these two arguments redundant? Why do you need to pass both?

Reply:
yeah, i will clean this up.